packages/webcomponents/src/components/order-terminals/order-terminals.tsx

```diff
@@ -119,61 +152,79 @@ export class OrderTerminals {
-            </div>
-            <div class="row">
-              <div class="col-6">
-                <h2 part={heading4}>Representative:</h2>
-                <ul class="list-group" part={listGroup}>
-                  <li class="list-group-item" part={listGroupItem}>Name: {this.business?.representative?.name}</li>
-                  <li class="list-group-item" part={listGroupItem}>Title: {this.business?.representative?.title}</li>
-                  <li class="list-group-item" part={listGroupItem}>Email: {this.business?.representative?.email}</li>
-                  <li class="list-group-item" part={listGroupItem}>Phone: {this.business?.representative?.phone}</li>
-                </ul>
+              <div class="row">
+                <h5 part={heading5}>Representative:</h5>
+                <div>
+                  <div>{this.business?.representative?.name}</div>
+                  <div>{this.business?.representative?.title}</div>
+                  <div>{this.business?.representative?.email}</div>
+                  <div>{this.business?.representative?.phone}</div>
+                </div>
               </div>
-              <div class="col-6">
-                <h2 part={heading4}>Shipping Address:</h2>
-                <ul class="list-group" part={listGroup}>
-                  <li class="list-group-item" part={listGroupItem}>Street: {this.business.legal_address?.line1}</li>
-                  <li class="list-group-item" part={listGroupItem}>{this.business.legal_address?.line2}</li>
-                  <li class="list-group-item" part={listGroupItem}>Postal Code: {this.business.legal_address?.postal_code}</li>
-                  <li class="list-group-item" part={listGroupItem}>City: {this.business.legal_address?.city}</li>
-                  <li class="list-group-item" part={listGroupItem}>State: {this.business.legal_address?.state}</li>
-                </ul>
+            </div>
+          )}
 
+          {this.terminalsIsLoading && (
+            <div class="mt-5">
+              <TerminalSelectorLoading />
+              <TerminalSelectorLoading />
+              <TerminalSelectorLoading />
+            </div>
+          )}
+
+          {!this.terminalsIsLoading && this.terminalModels && (
+            <div class="d-flex flex-column gap-2 mt-5">
+              {this.terminalModels.map((terminal) => (
+                <terminal-quantity-selector
+                  modelName={terminal.model_name}
+                  description={terminal.description}
+                  imageUrl={terminal.image_url}
+                  helpUrl={terminal.help_url}
+                  limit={this.orderLimit - this.order.totalQuantity}
+                  onSelectedQuantityChange={this.onSelectedQuantityChange.bind(this)}
+                />
+              ))}
+            </div>
+          )}
+
+          <div class="mt-3 text-end">Order limit: {this.orderLimit}</div>
+
+          {!this.businessIsLoading && this.business && this.shipping && (
```

Comment:
This is acceptable for right now - but we should consider how we can clean up these conditional loading states. Maybe using different render states to define when things like `this.business` and `this.terminalModels` are both finished loading. 

Like I said, fine for now because I know we wanna the MVP out the door, but something to think about when we have time to clean this up a little more! 🌟